types/Context.d.ts

```diff
@@ -62,6 +62,12 @@ export interface Context {
     res?: {
         [key: string]: any;
     };
+    /**
+     * If this flag is set to true in your function, error/warning logs for unrecommended `context.done()` patterns
+     * (e.g., using `context.done()` with `async`/`await`, calling `context.done()` twice) are suppressed.
+     * @default false
+     */
+    suppressBadPatternError?: boolean;
```

Comment:
That would make sense but the `suppressAsyncDoneError` would not cover the calling `context.done()` twice error. Or do we want this to suppress only the error in case of calling `context.done()` in an async function?

Reply:
I think it would cover it. It mentions "done", which can cover the multiple done case. We don't have to be as explicit as `suppressAsyncDoneOrDoneDoneError` lol. I just don't want a generic name. If you have other "specific" ideas feel free to suggest them - `suppressExtraDoneError` maybe?

Reply:
How does `suppressDoneErrors` sound?

Reply:
`suppressDoneErrors` still feels like you're going to suppress a bunch of different errors at once. Maybe I do want you to _only_ suppress the asyncDone error (aka not the `context.done` twice error). That's the only one affecting durable right?

Reply:
That is correct, that is the one affecting Durable

Reply:
Changed it so that this only suppresses the one error affecting durable